js/api/Forms.js

```diff
@@ -0,0 +1,33 @@
+import _ from 'lodash'
+import Parse from 'parse/react-native'
```

Comment:
package.json doesn't include the parse-react package I assume this is imported from.